Police-Data-Accessibility-Project/data-sources-app-v2
# Mc 485 typeahead location bug

#### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/485

#### Description

* Addresses bug causing documentation for `/typeahead/locations` endpoint to not display properly.
LIBTYFI
* Incorporated the `/typeahead/locations` endpoint into a new, consolidated schema configuration redesign.
* Removed tests that were more trouble than they were worth, breaking on irrelevant changes. 

#### Testing

* Run tests in `/tests` directory; confirm functionality
* Inspect endpoint documentation in API to ensure correct appearance.

#### Performance

* Removal of certain tests will improve test overhead marginally

#### Docs

* API documentation updated 

#### Breaking Changes

* No breaking changes.
